docs/REST API/rest-api.md

```diff
@@ -4301,7 +4301,12 @@ OS vulnerability data is currently available for Windows and macOS. For other pl
     {
       "id": 121,
       "name": "Google Chrome.app",
-      "package_available_for_install": "GoogleChrome.pkg",
+      "available_for_install": true,
+      "software_package": {
+        "name": "GoogleChrome.pkg",
+        "version": "125.12.0.3"
+      },
+      "app_store_app": null
```

Comment:
@mna We want to update this endpoint (`GET /hosts/:id/software`) to have similar behavior as GET /software/titles/:id - so adding available_for_install boolean + software_package for packages and app_store_app for VPP apps. One of these is always null.